app/src/main/java/org/simple/clinic/bp/entry/BloodPressureEntryUpdate.kt

```diff
@@ -43,6 +44,14 @@ class BloodPressureEntryUpdate : Update<BloodPressureEntryModel, BloodPressureEn
           ShowConfirmRemoveBloodPressureDialog((model.openAs as OpenAs.Update).bpUuid)
       )
 
+      is ScreenChanged -> next(model.screenChanged(event.type))
+
+      is BackPressed -> if (model.activeScreen == BP_ENTRY) {
+        dispatch<BloodPressureEntryModel, BloodPressureEntryEffect>(Dismiss)
```

Comment:
Use `dispatch(Dismiss as BloodPressureEntryEffect)` for better readability with the rest of the code block?

Reply:
Done